haanna/haanna.py

```diff
@@ -84,6 +95,14 @@ def set_preset(root, preset):
 
         return r.text
 
+    @staticmethod
+    def get_heating_status(root):
+        """Gets the active heating status"""
+        if root.find("appliance[type='heater_central']/logs/point_log/period/measurement").text == 'on':
```

Comment:
Add an extra filter to point_log to make sure the correct data is returned.
```suggestion
        if root.find("appliance[type='heater_central']/logs/point_log[type='central_heating_state']/period/measurement").text == 'on':
```